Cargo.toml

```diff
@@ -16,3 +16,14 @@ members = [
 	"types",
 ]
 resolver = "2"
+
+[workspace.package]
+authors = ["Parity Technologies <admin@parity.io>", "Pierre Krieger <pierre.krieger1708@gmail.com>"]
+edition = "2021"
+rust-version = "1.64.0"
+license = "MIT"
+repository = "https://github.com/paritytech/jsonrpsee"
+documentation = "https://docs.rs/jsonrpsee"
+homepage = "https://www.parity.io/"
+keywords = ["jsonrpc", "json", "http", "websocket", "WASM"]
+readme = "README.md"
```

Comment:
Any idea how it resolves this in dependencies; does it use the path relative to this workspace toml?

Reply:
According to https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table it should work

Reply:
Ah; readme and license paths are relative to the workspace root, ok cool! Not so useful if you want to have crate specific readme's but good if you just want to use the same one for all crates, so makes sense to me!